app-cdr/cddetect/cddetect-2.1.ebuild

```diff
@@ -1,17 +1,17 @@
-# Copyright 1999-2012 Gentoo Foundation
+# Copyright 1999-2019 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=4
+EAPI=7
+
 inherit toolchain-funcs
 
-DESCRIPTION="A tool for detecting the type of a CD/DVD without mounting it"
+DESCRIPTION="Tool for detecting the type of a CD/DVD without mounting it"
 HOMEPAGE="http://www.bellut.net/projects.html"
 SRC_URI="http://www.bellut.net/files/${P}.tar.gz"
 
 LICENSE="GPL-2"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
-IUSE=""
 
 S=${WORKDIR}
```

Comment:
...did you really test this ebuild after porting to EAPI 7?

Reply:
Huh... looks like i really forgot here.. I usually always test my changes of course - sorry for the mistake.